src/components/SelectSearch.tsx

```diff
@@ -45,58 +41,63 @@ export default class SelectSearch extends React.PureComponent<
         super(props);
 
         this.state = {
-            selection: props.initial
+            selections: props.initial
         };
 
-        this.selectRef = this.selectRef.bind(this);
-        this.loadOptions = this.loadOptions.bind(this);
-        this.onInputChange = this.onInputChange.bind(this);
-        this.onChange = this.onChange.bind(this);
+        bindCallbacks(this, {
+            include: ['selectRef', 'loadOptions', 'onChange']
+        });
     }
 
     private selectRef(ref: HTMLInputElement): HTMLInputElement {
         return (this.select = ref);
     }
 
+    public componentWillReceiveProps(nextProps: SelectSearchProps): void {
```

Comment:
ensures we're (a) updating state when we receive a new `initial` prop and (b) only doing so if it's a new value